pkg/core/state/notification_event.go

```diff
@@ -95,6 +168,10 @@ func (aer *AppExecResult) EncodeBinaryWithContext(w *io.BinWriter, sc *stackitem
 		aer.Events[i].EncodeBinaryWithContext(w, sc)
 	}
 	w.WriteVarBytes([]byte(aer.FaultException))
+	w.WriteVarUint(uint64(len(aer.Invocations)))
```

Comment:
You need to check if it's nil and encode only if it's not nil. It will save a single byte for cases when the node is running with SaveInvocations off.